pkg/controller/registry/reconciler/grpc.go

```diff
@@ -17,7 +17,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-const CatalogSourceUpdateKey = "catalogsource.operators.coreos.com/update"
+const (
```

Comment:
nit: why is this being changed?

Reply:
That was from moving around constants, reformatting that to the old form